pkg/sink/sink.go

```diff
@@ -141,7 +141,7 @@ func (r Sink) processTrigger(t *triggersv1.EventListenerTrigger, request *http.R
 	}
 	log := eventLog.With(zap.String(triggersv1.TriggerLabelKey, t.Name))
 
-	finalPayload, header, err := r.executeInterceptors(t, request, event, log)
+	finalPayload, header, err := r.ExecuteInterceptors(t, request, event, log)
```

Comment:
Any reason to expose this function outside :thinking: 
Because I don't see usage of `ExecuteInterceptors` anywhere other than `sink` package

Reply:
We'll eventually use this since the current processTriggerSpec does not call executeTrigger. But I think we can change the visibility in that PR!

Reply:
Ya i agree with you when we add changes then changing visibility that time make sense :+1: